README.markdown

```diff
@@ -1,9 +1,19 @@
-# Sphero.js
+# 🚫 DEPRECATED Sphero.js
 
-The official Orbotix JavaScript SDK module to programmatically control Sphero robots.
+This was the official Orbotix JavaScript SDK module to programmatically control Sphero robots manufactured through mid-2016. **This repo is no longer supported or maintained.**
 
 [![Build Status](https://travis-ci.org/orbotix/sphero.js.svg?branch=master)](https://travis-ci.org/orbotix/sphero.js) [![Code Climate](https://codeclimate.com/github/orbotix/sphero.js/badges/gpa.svg)](https://codeclimate.com/github/orbotix/sphero.js) [![Test Coverage](https://codeclimate.com/github/orbotix/sphero.js/badges/coverage.svg)](https://codeclimate.com/github/orbotix/sphero.js/coverage)
 
+
+*New SDKs have been produced for the [Sphero RVR](https://www.sphero.com/rvr). Links to the SDKs can be found below and more information can be found at [sdk.sphero.com](https://sdk.sphero.com/)*
+- [micro:bit MakeCode Extension](https://github.com/sphero-inc/sphero-sdk-microbit-makecode) (this can also be found in "Extensions" on the [MakeCode site](https://makecode.microbit.org/)!)
+- [micro:bit Python SDK](https://github.com/sphero-inc/sphero-sdk-microbit-python)
+- [Arduino Uno C++ SDK](https://github.com/sphero-inc/sphero-sdk-arduino-cpp)
+- [Arduino SpheroRVR Library](https://github.com/sphero-inc/sphero-sdk-arduino-cpp-library-manager) (this is the code for the Library that can be selected in the Arduino IDE under Sketch > Include Library)
```

Comment:
I don't think we want to direct people to the SpheroRVR github... it's nbd if they find it on their own but the only reason we really have it was to satisfy the requirements to get our SDK published, so maybe just make this a note after the sphero-sdk-arduino-cpp link (like you did for the micro:bit makecode one)